.github/workflows/run-frequent-coordination.yaml

```diff
@@ -0,0 +1,126 @@
+name: Run-Frequent-Coordination
+
+# Performs workloads we want to run as frequently as possible
+# Currently that is building/testing a variety of targets
+
+# Current compute time estimate:
+# ~116 hrs every 6 hours
+# There are 20 github runners, giving us a hard limit of 120 hrs every 6 hrs
+
+on:
+  schedule:
+    # Run 4 times a day
+    - cron: 0 0,6,12,18 * * *
+  workflow_call:
+    inputs:
+      gcchash:
+        description: 'GCC Hash'
+        required: true
+        type: string
+      issue_num:
+        description: 'Issue number of baseline'
+        required: true
+        type: string
+  workflow_dispatch:
+    branches:
+      - main
+    inputs:
+      gcchash:
+        description: 'GCC Hash'
+        required: true
+      multi_target:
+        description: 'Targets to run (libc:arch-abi;...)'
+        required: true
+      issue_num:
+        description: 'Bisection Issue Number (Optional)'
+        required: false
+
+jobs:
+  init-submodules:
+    uses: ./.github/workflows/init-submodules.yaml
+    with:
+      prefix: 'coord_'
+      gcchash: ${{ inputs.gcchash }}
+      branch: 'coordination'
+
+  creg: # Check Regressions. Short name so I can see the matrix string in github
+    needs: [init-submodules]
+    strategy:
+      fail-fast: false
+      matrix:
+        mode: [newlib, linux]
+        target:
+          [
+            rv32gc-ilp32d,
+            rv64gc-lp64d,
+            rv32gc_zba_zbb_zbc_zbs-ilp32d, # rv32 bitmanip
+            rv64gc_zba_zbb_zbc_zbs-lp64d, # rv64 bitmanip
+          ]
+        multilib: [non-multilib]
+    uses: ./.github/workflows/build-test.yaml
+    with:
+      mode: ${{ matrix.mode }}
+      target: ${{ matrix.target }}
+      gcchash: ${{ needs.init-submodules.outputs.gcchash }}
+      multilib: ${{ matrix.multilib }}
+      multitarget: ${{ github.event.inputs.multi_target }}
+      run_on_self_hosted: false
```

Comment:
Do we have the spare github runners to run this without self-hosted runners?

Reply:
Looking at yesterday's runs, we have 3 runs, each running for about 8 hrs. The start time was 20 hrs ago, 15 hrs ago, and 8 hrs ago. linux targets take around 5 hrs to complete while newlib takes around 3.5 hrs to complete. Adding a coordination branch would mean we'd have 8 runners that take 5 hrs to complete and 8 runners taking 3.5 hrs to complete for a total of 16 runners active. If we run at the same cadence as run-frequent, I think we could barely avoid having github runner resource errors